src/components/Modal/Modal.module.css

```diff
@@ -100,7 +101,6 @@
       display: flex;
       flex-direction: column;
       gap: var(--spacing-gap-xl, 24px);
-      overflow-y: auto;
```

Comment:
@ThisIsDemetrio recently made a change (https://github.com/mia-platform/design-system/pull/619) to make the sider properly scroll, is this change still compatible with the sider scrolling?

Reply:
after we sort this out LGTM :)

Reply:
I talked with @ThisIsDemetrio and everything is fine